daemon/src/connection.rs

```diff
@@ -429,7 +429,7 @@ impl Actor {
 
         let msg_name = msg.name();
 
-        tracing::trace!(target: "wire", "Received {msg_name}", );
+        tracing::trace!(target: "wire", msg_name = msg_name, "Received");
```

Comment:
nit: `msg_name` probably doesn't need to be repeated, as we're not renaming it here